docs/examples/data-completeness.py

```diff
@@ -0,0 +1,85 @@
+"""
+Missing Data Periods
+====================
+
+Identifying days with missing data using a "completeness" score metric,
+and filtering out consecutive days with a low completeness score for
+future data analysis.
+"""
+
+# %%
+# Identifying days with missing data and filtering these days out reduces noise
+# when performing data analysis. This example shows how to use a
+# daily data "completeness" score to identify and filter out days with missing
+# data. This includes using
+# :py:func:`pvanalytics.quality.gaps.completeness_score`,
+# :py:func:`pvanalytics.quality.gaps.complete`, and
+# :py:func:`pvanalytics.quality.gaps.trim_incomplete`.
+
+import pvanalytics
+from pvanalytics.quality import gaps
+import matplotlib.pyplot as plt
+import pandas as pd
+import pathlib
+
+# %%
+# First, we import the AC power data stream that we are going to check for
+# completeness. The time series we download is a normalized AC power time
+# series from the PV Fleets Initiative, and is available via the DuraMAT
+# DataHub:
+# https://datahub.duramat.org/dataset/inverter-clipping-ml-training-set-real-data
+
+pvanalytics_dir = pathlib.Path(pvanalytics.__file__).parent
+file = pvanalytics_dir / 'data' / 'ac_power_inv_2173.csv'
+data = pd.read_csv(file, index_col=0, parse_dates=True)
+data = data.asfreq("15T")
+
+# %%
+# Now, use :py:func:`pvanalytics.quality.gaps.completeness_score` to get the
+# percentage of daily data that isn't NaN.
+data_completeness_score = gaps.completeness_score(data['value_normalized'])
```

Comment:
Maybe add a few comments here about what the data are, e.g., what is 'value_normalized'? Is it any column of numbers or are there constraints? I think one assumption that is not so obvious is that the completeness score is a fraction of a 24 hour day, thus nighttime values are expected.